resources/definitions/mixware_wand.def.json

```diff
@@ -0,0 +1,194 @@
+{
+  "name": "Wand",
+  "version": 2,
+  "inherits": "fdmprinter",
+  "metadata": {
+    "visible": true,
+    "author": "Mixware",
+    "manufacturer": "Mixware",
+    "file_formats": "text/x-gcode",
+    "has_materials": true,
+    "has_machine_quality": true,
+    "preferred_material": "mixware_pla",
+    "preferred_quality_type": "coarse",
+    "has_variants": false,
+    "platform": "mixware_wand_platform.STL",
+    "machine_extruder_trains": {
+      "0": "mixware_wand_extruder_0"
+    }
+  },
+  "overrides": {
+    "machine_name": {
+      "default_value": "Wand"
+    },
+    "machine_start_gcode": {
+      "default_value": "G28 ;Home\n;Prime the extruder\nM117 Print\nG92 E0\nG1 F1500 X-2 Y0 Z0.28\nG1 F2400 X-2 Y70 Z0.28 E8\nG1 F1500 X-1.7 Y70 Z0.28\nG1 F2400 X-1.7 Y0 Z0.28 E16\nG92 E0\nG92 E0\nG1 Z0.18 F2400 ;Move Z Axis up"
+    },
+    "machine_end_gcode": {
+      "default_value": "M104 S0\nM140 S0\n;Retract the filament\nG92 E1\nG1 E3 F300\nG1 F1500 Z100\nG28 X0 Y0\nM25\nM84"
+    },
+    "material_diameter": {
+      "default_value": 1.75
+    },
+    "machine_width": {
+      "default_value": 102
+    },
+    "machine_depth": {
+      "default_value": 102
+    },
+    "machine_height": {
+      "default_value": 102
+    },
+    "machine_nozzle_size": {
+      "default_value": 0.4
+    },
+    "layer_height": {
+      "minimum_value": "0.05",
+      "maximum_value": "0.2"
+    },
+    "wall_thickness": {
+      "default_value": 0.8,
+      "minimum_value": "0.4",
+      "maximum_value": "1.6"
+    },
+    "top_bottom_thickness": {
+      "default_value": 1.0,
+      "minimum_value": "0",
+      "maximum_value": "2"
+    },
+    "optimize_wall_printing_order": {
+      "default_value": true,
+      "settable_per_mesh": false
+    },
+    "z_seam_type": {
+      "default_value": "back"
+    },
+    "z_seam_y": {
+      "value": 99
+    },
+    "material_print_temperature": {
+      "default_value": 200,
+      "minimum_value": "180",
+      "maximum_value": "220"
+    },
+    "material_bed_temperature": {
+      "default_value": 0
+    },
+    "retraction_enable": {
+      "default_value": true
+    },
+    "retraction_amount": {
+      "default_value": 4,
+      "minimum_value": "2",
+      "maximum_value": "5.5"
```

Comment:
As before. Are you very sure that you want min/max values set? As a rule we only use min/max error values if it is very likely to cause damage to the machine or is actually impossible. If we want to indicate that it might result in poorer prints, we use minimum_warning_value. 

Please check this for all the settings that you added.